src/Tools/CodeGenerator/Featurizers.yaml

```diff
@@ -1414,58 +1414,57 @@ featurizers:
     num_output_columns: 1
     has_dynamic_output: true
     description: |-
-      Computes lags and leads for values. Works for general time series data sets.
-      For sparse time series data sets, forward fill imputer will be applied so input time series would always be continuous
-      and have a consistent time interval between rows.
+      Copying values from back or in the future per grain. Works for general time series data sets.

```

Comment:
Now that we are removing the time information, this will work with any sorted dataset and not just timeseries datasets. I would also change the language from "back" to something else if possible. I think "back" isn't very clear. Maybe "past"? Or "prior"? What do you think?

Reply:
I like "prior".

Reply:
Makes sense. Changed to "prior".